### ISSUE TITLE
Path parsing in "parse_data_NSx"

### ISSUE THREAD

**Issue by aboharbf** [csn-le/wave_clus#108]
Putting in a path of the following sort

'D:/Folder/Folder/Folder/Folder/Folder/Folder/File.ns5'

Triggers the filename changing conditional in "parase_data_NSx.m", despite the fact the path is valid. I fixed it by adding a new "&& ~strcmpi(filename(3),'/')" after the || which triggers it.

**Comment by ferchaure** [comment]
I don't follow you, is that a Windows machine with a character '/' as the folder separator?

**Comment by aboharbf** [comment]
I figured it had something to do with that. I work with folks who use the UNIX/Mac convention of slashes, but since MATLAB is able to flip all the slashes when it reads paths (as long as they are all the same) I've simply stuck with the convention of using that slash in everything I write which may interface with our lab's software suite (phyzzy, its on github). I'm currently working on integrating wave_clus into this suite, so the paths being fed to it look this way. I realize that may limit the generalizability of such an addition to the code.

**Comment by ferchaure** [comment]
I understand now, I will add this case in the next commit.

**Comment by aboharbf** [comment]
Agreed. It seems like there is also the function "contains" introduced in 2016b which may be useful (though i realize desires for backwards compatibility make this unwise.)
